redirect/routes.php

```diff
@@ -171,7 +171,15 @@ class Route {
 	'pgm' => [
 	      'description' => "Probabilistic Graphical Models in Computer Vision",
 	      'target' => 'http://vision.in.tum.de/teaching/ss2017/pgmcv'
-	]
+	],
+        'eraprak' => [
```

Comment:
anyone got a better abbreviation? there is no era this term but I'd still avoid changing between normal lecture and the "praktikum" each term